model.py

```diff
@@ -6520,13 +6567,15 @@ def lookup_by_temporary_token(cls, _db, data_source, type, token):
         return credential
 
     @classmethod
-    def temporary_token_create(self, _db, data_source, type, patron, duration):
+    def temporary_token_create(
+            self, _db, data_source, type, patron, duration, value=None
+    ):
```

Comment:
I think I originally didn't use this because it seemed like the temporary tokens were single-use - lookup_by_temporary_token expires them. Maybe we could rename that method.